public/assets/js/index.js

```diff
@@ -14,15 +14,11 @@ function loadAndRenderItems () {
 loadAndRenderItems()
 
 /**
- * Anytime there is a data change we reload and render the list of items
  */
-hoodie.store.on('change', loadAndRenderItems)
-hoodie.store.on('clear', function () {
-  render([])
-})
-
```

Comment:
`hoodie.store.on('change')` gets emitted for both, local and remote changes. Now that you removed it, the table will not get updated for remote changes. You can try it out by signing in with the same account in two different browsers or in normal and private mode. On https://tracker.hood.ie/ the changes will sync, with your solution they won’t any more.

I know that the current solution is not ideal, but what we optimize for is simplicity of code. We try to keep the barriers to the absolute minimum, so we refrain from using any 3rd party libraries, not even jQuery. I’d be happy to fix this issue, but only if it does not add too much complexity. I know that is quite a challenge :/